index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Bangtan World Tour</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="wrapper">
+        <!---------------- HEADER ---------------->
+        <header class="header-content">
+            <div class="header-desktop1">
+                <nav>
+                    <ul>
+                        <li><img src="./assets/img/facebook.png" alt="facebook logo"></li>
+                        <li><img src="./assets/img/twitter.png" alt="twitter logo"></li>
+                        <li><a href="#">English</a></li>
+                        <li><a href="#">My Page</a></li>
+                        <li><a href="#">Logout</a></li>
+                    </ul>
+                </nav>
+            </div>
+            <button class="hamburger-btn"></button>  
+            <div class="header-two">
+                <img class="logo" src="./assets/img/logo.png" alt="Group logo">
+                <img class="close-btn" src="assets/img/close-icon.png" alt="">
+                <nav class="desktop-menu">
+                    <ul>
+                        <li class="nav-item"><a class="menu-item" href="./about.html">About</a></li>
+                        <li class="nav-item"><a class="menu-item" href="#Program">Program</a></li>
+                        <li class="nav-item"><a class="menu-item" href="#">Join</a></li>
+                        <li class="nav-item"><a class="menu-item" href="#">Sponsor</a></li>
+                        <li class="nav-item"><a class="menu-item" href="#">News</a></li>
+                        <li class="nav-item1"><a class="concert" href="#"></a>Spring Day World Tour</li>
+                    </ul>
+                </nav>
+            </div>
+        </header>
+        <!------------------ HOME ------------------>
+        <section class="home">
+            <div class="presentation">
```

Comment:
- [ ] Your home page heading section is well designed 👏  But I notice that you use different colors for the text and also some element font_size are not as on the [ the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). Please go through it and make sure to use the exact same colors, for text, backgrounds, borders, etc, from the original design as required for this project 👍